libs/langchain/langchain/vectorstores/scann.py

```diff
@@ -23,9 +22,9 @@ def normalize(x: np.ndarray) -> np.ndarray:
     return x
 
 
-def dependable_scann_import() -> Any:
+def _dependable_scann_import() -> Any:
```

Comment:
all of the dependable...->_dependable... changes are technically breaking changes and imo not worth it (even though in an ideal world i agree they'd be private)

Reply:
Those "import checking" functions are spread all across LC. We even have a generic function for it (but nobody uses it). 
They are really exposing implementation details for users and should not be there in most cases. To me, exposing the internals is more like a bug. 
Anyway, I'll rollback these function names.